src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -94,38 +121,31 @@ export class Main extends React.Component {
   }
 }
 
-const mainSelector = createSelector(
```

Comment:
I removed this selector because it felt confusing and it took me a long time to figure out what properties from our Redux state this component was using as a result